src/components/Thought.js

```diff
@@ -224,7 +224,7 @@ const canDrop = (props, monitor) => {
   const thoughtsTo = props.thoughtsRankedLive
   const thoughts = pathToContext(props.thoughtsRankedLive)
   const context = contextOf(thoughts)
-  const isSorted = getSortPreference(state, context) === 'Alphabetical'
+  const isSorted = getSortPreference(state, context).includes('Alphabetical')
```

Comment:
Can you explain this? The short preference should exactly match `Alphabetical`.

Reply:
@raineorshine It returns array, so, it couldn't match and this check never was true.

Reply:
I would expect it to return a `string` value, since the sort preference is indicated by a string like “Alphabetical”. Wouldn’t you agree? If it is returning an array than it has a bug.

Reply:
@raineorshine Yes, I agree. But I'm not sure if it affect other places where it's used. So, I just used `includes` instead of changing `getSortPreference `.

Reply:
What is the faulty condition that is causing `getSortPreference` to return an array? That should be investigated. We want to get to the root of the problem rather than covering it up.

Reply:
Typescript would throw an error if any of the callers were expecting an array. There are only seven places in the codebase where `getSortPreference` is called, so it is easy to confirm by hand.

Reply:
@raineorshine I was written on JS at first. I got TS version only during merge. In JS version there are no types. So, I thought that there can be a list of sort preferences as it also make sense if we will be able to sort not only alphabetically.

Reply:
Okay, that makes sense. We don't have that now, but maybe in the future.